askama_parser/src/node.rs

```diff
@@ -311,8 +311,15 @@ impl<'a> Node<'a> {
                 cut(pair(ws(identifier), opt(Whitespace::parse))),
             ))),
         ));
-        let (i, (pws, _, (name, _, (scope, nws)))) = p(i)?;
-        Ok((i, Self::Import(Ws(pws, nws), name, scope)))
+        let (i, (pws, _, (path, _, (scope, nws)))) = p(i)?;
```

Comment:
If we're going to do this I think we should also move the parser into an `Import::parse()` constructor, with `Node::import()` only wrapping the result of that in `Self::Import`.